ids/mapping.json

```diff
@@ -2,5 +2,6 @@
   "ankush": "ankushdharkar",
   "nikhil": "whydonti",
   "sahil": "invalidtoken",
-  "sumitdhanania": "sumitd94"
+  "sumitdhanania": "sumitd94",
+  "sheena": "sheenachhabradev"
```

Comment:
Alphabetically incorrect. Please fix this

Reply:
Fix - updated in alphabetical order.